vtex-apps/lightspeed
# [LIGHTSPD-38] fix setComplete

**What problem is this solving?**

The bug was marked completed order as incomplete, this PR fixed `setComplete` function to mark completed order as complete

**How should this be manually tested?**

Go to site [https://aurora--benscycle.myvtex.com/admin/lightspeed](url), you should not see any empty row. And if you do, retry the order should fix it and mark it to complete
